testsuite/integration/basic/pom.xml

```diff
@@ -3069,6 +3069,7 @@
                                     <excludes>
                                         <!-- JpaJarFileTestCase requires custom hibernate module -->
                                         <exclude>org/jboss/as/test/integration/jpa/jarfile/JpaJarFileTestCase.java</exclude>
+                                        <exclude>org/jboss/as/test/integration/ee/concurrent/scheduled/ExceptionInScheduleTestCase.java</exclude>
```

Comment:
This test shouldn't need to be excluded. If it is excluded it needs a comment explaining why.

Perhaps remove the @Remote from ExecNumber? The test runs in container and shouldn't need @Remote.